app/managers.py

```diff
@@ -0,0 +1,43 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            """INSERT INTO actors (first_name, last_name)
+            VALUES (?, ?)""",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            """SELECT * FROM actors"""

```

Comment:
even if it fits in one line, sql standard of formatting suggests to start key words from new line. Also is common to put triple quotes on a separate line like so:
```
"""
SELECT * 
FROM actors
"""

 Please check in other queries as well if formatiing style is in compliance